Inventory.cs

```diff
@@ -1,176 +1,121 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace JanasInventoryManagementSystem
+﻿namespace JanasInventoryManagementSystem
 {
     public class Inventory
     {
-        private static List<Product>? products = new List<Product>();
+        private readonly Dictionary<int, Product> _products = new();
+        DataManager dataManager = new();
+        ProgramHelper programHelper = new();
 
-        public void Add()
+        public void Add(Product product)
```

Comment:
Could you please create an interface 
IInventory and make your class implement it